src/connectors/hierarchical-menu/connectHierarchicalMenu.js

```diff
@@ -232,6 +232,7 @@ export default function connectHierarchicalMenu(renderFn, unmountFn = noop) {
         return {
           items,
           refine: this._refine,
+          canRefine: items.length > 0,
```

Comment:
should be > 1, and possibly same for breadcrumb?

Reply:
Even if there is only one item, it doesn't mean it's already selected. So, I think canRefine should be true. What do you think?

Reply:
yes, but it won't make a difference wether it's selected or not. If all items are of a single category, selecting it won't change the results.

Reply:
Yes, that's correct. This and [that](https://github.com/algolia/instantsearch.js/pull/4691) make sense, but it will deviate from the implementation of Vue InstantSearch. Should we deviate now and later introduce a breaking change in Vue InstantSearch when it starts using the canRefine from InstantSearch.js?

Reply:
Yes, I think it's better to have a small breaking change in Vue InstantSearch later when we use canRefine there :)

Reply:
Thinking again, another point to consider is, when this widget is wrapped with `panel`, it could collapse or hide the widget when there is only one item if developers use `canRefine` as the condition. Would you expect it to be hidden with only item? or with zero item?

Reply:
I would expect it to be hidden when there's no difference possible anymore in the results, therefore 1 item would do that :)